.github/workflows/oppiabot.yml

```diff
@@ -4,13 +4,22 @@ on:
   issues:
     types:
       - labeled
+  pull_request_target:
+    branches:
+      - develop
+      - release-*
 
 jobs:
   oppiabot:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Ensure that appropriate labels are added to issues
-        uses: oppia/oppiabot@1.1.0
+      - name: Github Actions from Oppiabot
+        uses: oppia/oppiabot@1.2.0
         with:
           repo-token: ${{secrets.GITHUB_TOKEN}}
+        env:
+          SHEETS_TOKEN: ${{ secrets.SHEETS_TOKEN }}
+          SHEETS_CRED: ${{ secrets.SHEETS_CRED }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}`
+          SPREADSHEET_ID: ${{ secrets.SPREADSHEET_ID }}
```

Comment:
Are we going to pass these secret values to a third-party script? (i.e, a script that doesn't exist in this repo?)

Reply:
No these values will be present as environment variables in the Ubuntu OS in the Github Action. To be clear, Github action and the script in the develop branch will be able to access it.